ophyd/sim.py

```diff
@@ -138,7 +138,6 @@ def __init__(self, func=None, *,
             connected=True,
         )
 
-        self.log.info(self)
```

Comment:
This one explains a lot of the startup logging spam

Reply:
Yep! I have a bet with the author of this line that if it ever comes up that this would have bailed us out of a hard-to-debug situation I owe a nice gift of some kind. We'll see!